web/src/interfaces.ts

```diff
@@ -84,6 +84,12 @@ export interface Listing extends ListingPayload {
   listingStatus: ListingStatus;
 }
 
+/**
+ * ListingQuery Interface that contains the fields of the query that
+ * would be sent to the server to query for commits.
```

Comment:
query for listings

Reply:
@ifhuang Oops it slipped my mind to change it! Will change!